packages/dx-demo-shell/generate-demos.js

```diff
@@ -192,26 +199,42 @@ const generateDemoRegistry = () => {
     );
   });
 
+  let returnedString;
+  const productName = getCurrentProductName();
+
+  if (attachDemo) {
+    returnedString = (themesAcc, themeName, demoSource, fileName) => (
+      `${themesAcc}\n${indent(`'${themeName}': {\n`
+      + `  demo: require('.${fileName}').default,\n`
```

Comment:
It's better to extract the `demo: require...` string to a variable depending on a `attachDemo` value to reduce duplication

Reply:
Done